sources/packages/backend/workflow/test/test-utils/factories/assessment-consolidated-data.ts

```diff
@@ -97,5 +99,19 @@ function setDefaultAssessmentConsolidatedData(): AssessmentConsolidatedData {
     partner1StudentStudyWeeks: null,
     partner1TotalIncome: null,
     partner1CRAReportedIncome: null,
+    assessmentId: null,
+    applicationExceptionStatus: null,
   } as AssessmentConsolidatedData;
 }
+
+export function createFakeConsolidatedFulltimeData(
+  programYear: string,
+): AssessmentConsolidatedData {
+  const [, programEndYear] = programYear.split("-");
+  const assessmentConsolidatedData =
```

Comment:
I have a qn here, next if we want to test a different set of info, for eg, TEST a "full time" assessment with a "dependent" status application, are we planning to make this method generic for all FT tests or are we planning to create "createFakeConsolidatedSOMETHINGFT". do we have a plan? or we are planning to figure it out on the way.

Reply:
The plan should be for e.g. TEST a "full time" assessment with a "dependent" if required only at a test case level, then it will be prepared at test case by using generic method and modifying it. Same if it is required at e2e test file level, then it can be prepared at beforeAll(), or else if it is required for multiple test files then createFakeConsolidatedSOMETHINGFT will be created at factory level.

Reply:
Also, there is nothing against creating a new `createFakeAssessmentConsolidatedData` with some initial data as a baseline, for instance.
- `createFakeAssessmentConsolidatedDataForTwoParents`
- `createFakeAssessmentConsolidatedDataWithDependents`
- `createFakeAssessmentConsolidatedDataWithPartner`
- `createFakeAssessmentConsolidatedDataForPIR`
Or have all of them as options on the `createFakeAssessmentConsolidatedData`
```ts
createFakeAssessmentConsolidatedData(options: {
    includeParent1SupportingData: boolean,
    includeParent2SupportingData: boolean,
    includePartnerSupportingData: boolean,
    generateDependents: numbers
})
```